blocks/library/chat/chat-transcript.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { uniq } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import classnames from 'classnames';
+
+export function normalizeTranscript( value ) {
+	const transcript = {
+		authors: [],
+		messages: [],
+	};
+
+	if ( ! value ) {
+		return transcript;
+	}
+
+	let authorIndex = -1;
+
+	value.split( '\n' ).map( ( line ) => {
+		line = line.trim();
+
+		const messageParts = line.split( ':' );
```

Comment:
This should add a `limit` param, so like `line.split( ':', 2 )`. Otherwise it will choke on a line like:

> John: I really like the ratio 1:1.